examples/resnet50_trt/Readme.md

```diff
@@ -0,0 +1,43 @@
+Description:
+This is an example to show how to use Python Backend to preprocess your image before using TensorRT to do inference. There will be an ensemble model include preprocess and resnet50_trt.
```

Comment:
"This example shows how to preprocess your inputs using Python backend before it is passed to the TensorRT model for inference."